pkg/proto/components/v1/state.pb.go

```diff
@@ -1419,18 +1419,66 @@ func (*BulkDeleteResponse) Descriptor() ([]byte, []int) {
 	return file_dapr_proto_components_v1_state_proto_rawDescGZIP(), []int{20}
 }
 
+type BulkGetRequestOptions struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Parallelism int64 `protobuf:"varint,1,opt,name=parallelism,proto3" json:"parallelism,omitempty"`
```

Comment:
Yes: https://github.com/dapr/components-contrib/blob/master/state/bulk.go#L22-L27

Although components that implement native BulkGet can decide to ignore that (because they will perform a single round-trip call to the DB).